specification/cognitiveservices/data-plane/Speech/SpeechToText/preview/v3.1-preview.1/speechtotext.json

```diff
@@ -0,0 +1,6237 @@
+        },
+        "type": {
+          "description": "The type of this component.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "Dataset": {
+      "title": "Dataset",
+      "required": [
+        "displayName",
+        "kind",
+        "locale"
+      ],
+      "type": "object",
+      "properties": {
+        "links": {
+          "$ref": "#/definitions/Links"
+        },
+        "properties": {
+          "$ref": "#/definitions/DatasetProperties"
+        },
+        "kind": {
+          "$ref": "#/definitions/DatasetKind"
+        },
+        "self": {
+          "format": "uri",
+          "description": "The location of this entity.",
+          "type": "string",
+          "readOnly": true
+        },
+        "displayName": {
+          "description": "The display name of the object.",
+          "type": "string"
+        },
+        "description": {
+          "description": "The description of the object.",
+          "type": "string"
+        },
+        "project": {
+          "$ref": "#/definitions/EntityReference"
+        },
+        "contentUrl": {
+          "format": "uri",
+          "description": "The URL of the data for the dataset.",
+          "type": "string"
+        },
+        "customProperties": {
+          "description": "The custom properties of this entity. The maximum allowed key length is 64 characters, the maximum\r\nallowed value length is 256 characters and the count of allowed entries is 10.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "locale": {
+          "description": "The locale of the contained data.",
+          "type": "string"
+        },
+        "lastActionDateTime": {
+          "format": "date-time",
+          "description": "The time-stamp when the current status was entered.\r\nThe time stamp is encoded as ISO 8601 date and time format\r\n(\"YYYY-MM-DDThh:mm:ssZ\", see https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations).",
+          "type": "string",
+          "readOnly": true
+        },
+        "status": {
+          "$ref": "#/definitions/Status"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "The time-stamp when the object was created.\r\nThe time stamp is encoded as ISO 8601 date and time format\r\n(\"YYYY-MM-DDThh:mm:ssZ\", see https://en.wikipedia.org/wiki/ISO_8601#Combined_date_and_time_representations).",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "DatasetKind": {
+      "title": "DatasetKind",
+      "description": "Type of data import.",
+      "enum": [
+        "Language",
+        "Acoustic",
+        "Pronunciation",
+        "AudioFiles",
+        "LanguageMarkdown"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "DatasetKind",
+        "modelAsString": false,
```

Comment:
The fact that this was modelAsString:false, it is adding a new field *and* it is still specified as modelAsString:false is troubling.At the very least, it should be marked as modelAsString:true since we have proven that fields will be added over time.